### PR TITLE
Ad admin scripts

### PR DESCRIPTION
**Pull request by Vojtech-Sassmann**
* Added scripts for ad_admin_user_mu_ucn and ad_admin_group_mu_ucn services.
* These services are used to manage privileged users and their groups in the ad. 
* Privileged users are users with the `mu-adm` login.

### CODE REVIEW

File: gen/ad_admin_user_mu_ucn [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+	}
+
+	# There MUST be an empty line after each entry, so entry sorting and diff works on slave part
+	print FILE "\n";
+
+}
+
+close(FILE);
+
+sub checkBase64 {
```
Comment (balcirakpeter) [inline gen/ad_admin_user_mu_ucn:118]: Not used subroutine

File: send/ad_admin_group_mu_ucn [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+
+			# Focus only on users
+			#@to_be_removed = grep(/OU=(Users).*/, @to_be_removed);
+			my $response_remove = remove_members_from_entry($ldap, $service_name, $ad_entry, \@to_be_removed);
+			unless ($response_remove == $SUCCESS) {
+				print STDERR "Failed to remove all members from group during the group removal process: ".$ad_entry->dn() . "\n";
+				ldap_log($service_name, "Failed to remove all members from group during the group removal process: ".$ad_entry->dn());
+				$counter_group_not_emptied++;
+			}
+			# NOT REMOVING GROUPS IN UCN AD
```
Comment (balcirakpeter) [inline send/ad_admin_group_mu_ucn:180]: You can remove this commented part

File: send/ad_admin_group_mu_ucn [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+	foreach my $ad_entry (@ad_entries) {
+		my $cn = $ad_entry->get_value('cn');
+		unless (exists $perun_entries_map{ lc $cn }) {
+
+			# clear members
+			# load members of a group from AD
+			my @to_be_removed = load_group_members($ldap, $ad_entry->dn(), $filter);
+
+			# Focus only on users
+			#@to_be_removed = grep(/OU=(Users).*/, @to_be_removed);
```
Comment (balcirakpeter) [inline send/ad_admin_group_mu_ucn:173]: You can remove the comment

File: send/ad_admin_user_mu_ucn [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+sub process_disable() {
+	foreach my $entry (@ad_entries) {
+
+		# We create only entries for normal UČO users
+		my $login = $entry->get_value('cn');
+		next if exists $perun_entries_map{$login};
+
+		my $entry_uac = $entry->get_value('userAccountControl');
+		if (is_uac_enabled($entry_uac) == 1){
+			#print "is_uac_enabled == 1 for user: " . $login . "\n";
```
Comment (balcirakpeter) [inline send/ad_admin_user_mu_ucn:109]: You can remove the comment

**Review by balcirakpeter — CHANGES_REQUESTED** [review]

**Review by balcirakpeter — APPROVED** [review]

**Review by metodej — APPROVED** [review]

**Pull request merged**